### PR TITLE
Add test for PDF010

### PR DESCRIPTION
**Pull request by mgmanzella**
Closes https://github.com/MarcoGorelli/pandas-dev-flaker/issues/12

### CODE REVIEW

File: tests/conftest_test.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        )
+    }
+
+
+@pytest.mark.parametrize(
+    "source",
+    (
+        pytest.param(
+            "conftest = 'foo'",
+            id="set conftest",
```
Comment (MarcoGorelli) [inline tests/conftest_test.py:25]: assign to conftest?

File: tests/conftest_test.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+@pytest.mark.parametrize(
+    "source, expected",
+    (
+        pytest.param(
+            "import conftest",
+            "1:0: PDF010 import from 'conftest' found",
+            id="import from conftest",
+        ),
```
Comment (MarcoGorelli) [inline tests/conftest_test.py:40]: nice! Can you also add a testcase which has `from conftest import foo`? (this one may not pass... :flushed: )

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: tests/conftest_test.py (hunk 2 above)
Comment (MarcoGorelli) [reply tests/conftest_test.py:40]: this one likely passes, but `from pandas.conftest import foo` likely won't - can you include that one as well? If you know how to fix the source code to make it pass, then even better :+1:

Comment (mgmanzella) [reply tests/conftest_test.py:40]: yes for sure!

File: tests/conftest_test.py (hunk 1 above)
Comment (mgmanzella) [reply tests/conftest_test.py:25]: 👍

File: tests/conftest_test.py (hunk 2 above)
Comment (mgmanzella) [reply tests/conftest_test.py:40]: `from pandas.conftest import foo` didn't pass -- should i open an issue for that case separately?

Comment (MarcoGorelli) [reply tests/conftest_test.py]: sure - let's take this one as it is, and if you're interested in making a separate PR to make `from pandas.conftest import foo` pass as well, then even better! (else I'll do it later)

Comment (mgmanzella) [reply tests/conftest_test.py]: yeh that's what i was getting at 🤣  i wanted to work on making tests pass for that case

Comment (MarcoGorelli) [reply tests/conftest_test.py]: :laughing: cool - I went and opened a new issue anyway, #16

**Pull request merged**